spec/support/fetchable_behavior.rb

```diff
@@ -96,6 +99,40 @@
         expect(described_class.create!(valid_attributes.merge(url: url)).url).to eq("http://www.nps.gov/hey/I/am/usagov/and/love/extra/slashes.shtml")
       end
     end
+
+    context "when URL doesn't have a protocol" do
+      let(:url) { "www.nps.gov/sdfsdf" }
+
+      it "should prepend it with https://" do
```

Comment:
This isn't your fault, as this spec was copied, but let's tweak this to [avoid using should](http://www.betterspecs.org/#should):
`it 'prepends it with "https://"'`